### ISSUE TITLE
poly_heat.m in the Matlab folder

### ISSUE THREAD

**Issue by km-git-acc** [km-git-acc/dbn_upper_bound#21]
Have added Gabriel Peyre's matlab file poly_heat.m to the matlab folder. That folder is currently empty so no possible conflicts and its available in the master branch.

**Comment by sujitnair** [comment]
Thanks, @km-git-acc

**Comment by sujitnair** [comment]
I also wrote an email to Gabriel. It will be great if he decides to be a collaborator here!

**Comment by sujitnair** [comment]
@km-git-acc 
https://github.com/gpeyre

**Comment by km-git-acc** [comment]
Yeah. He would also be a good mentor to have. I have sent him an invite.
